### PR TITLE
Update contributor guideline

### PR DESCRIPTION
**Pull request by yuanming-hu**
### Purpose

- No issue is linked to this PR.

### Changes

- No changes.

### Review Instructions

- No instructions.

### CODE REVIEW

File: website/docs/lang/articles/contribution/index.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
...
 ---
 
-# Contribution Guide
+# Contribution guide
 
 Welcome to the Taichi community! We really appreciate your contribution!
 
-:::caution reminder
+:::tip reminder
 Please take some time to familiarize yourself with this contribution guide before making any changes.
```
Comment (rexwangcc) [inline website/docs/lang/articles/contribution/index.md:10]: Circling back on this page, I start to feel like it's somewhat redundant... I meant to use it as a "Index page" for all kinds of contributions but it overlaps with the `contributor_guide`, maybe we could merge this with the other page? (i.e. move the `tip` block into `contributor_guide.md` and remove this page), wdyt?

**Review by rexwangcc — APPROVED** [review]
I left one nit but in general LGTM, thanks!!

File: website/docs/lang/articles/contribution/contributor_guide.md [hunk 2]
```diff
@@ -365,24 +357,15 @@ For more details on how to write a test case, see
...
-- We use [reStructured
-  text](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html)
-  (.rst) to write documentation.
-- We host our documentation online using
-  [readthedocs.io](https://taichi.readthedocs.io/en/stable).
-- Use `ti doc` to build the documentation locally.
-- Open the documentation at `docs/build/index.html`.
+Documentation source files are under the `docs/` folder of [**the main Taichi repo**](https://github.com/taichi-dev/taichi).
+An automatic service deploys updated documentation at
+[the Taichi documentation site](https://docs.taichi.graphics) ([repo](https://github.com/taichi-dev/docs.taichi.graphics)).
```
Comment (rexwangcc) [inline website/docs/lang/articles/contribution/contributor_guide.md:362]:
```suggestion
An automatic service syncs the updated content with our [documentation repo](https://github.com/taichi-dev/docs.taichi.graphics) and deploys the documentation at [the Taichi documentation site](https://docs.taichi.graphics).
```

File: website/docs/lang/articles/contribution/index.md (hunk 1 above)
Comment (yuanming-hu) [reply website/docs/lang/articles/contribution/index.md:10]: Sounds good! Let me move this tip to contributor_guide.md :-)

**Review by FantasyVR — APPROVED** [review]
LGTM

**Pull request merged**